package.json

```diff
@@ -55,6 +55,7 @@
     "@types/node": "~18.7.23",
     "@types/prettier": "~2.7.1",
     "@typescript-eslint/eslint-plugin": "~5.38.1",
+    "@typescript-eslint/experimental-utils": "^5.38.1",
```

Comment:
Why is this needed?

Reply:
I can't remember why ahah. I'm sorry for the delay @Shinigami92 I have a lot of stuff to do right now 🥲

I think I'll get back to the project in a week or two and resume theses PR

However, what I wanted to do was to finish our patch module, and once merged, regenerate the types for the plugins I proposed PRs for